app/models/green_lanes/categorisation.rb

```diff
@@ -31,6 +31,18 @@ def load_from_string(data)
       def all
         @all ||= load_from_file
       end
+
+      def filter(regulation_id:, measure_type_id:, geographical_area: nil)
+        return [] if regulation_id.blank? || measure_type_id.blank?
+
+        all.select { |c| c.match?(regulation_id:, measure_type_id:, geographical_area:) }
```

Comment:
super minor suggestion but you could just use `...` to pass all params through?

Reply:
in this case I'd prefer to be more explicit